app/modules/notifications/models.py

```diff
@@ -155,26 +151,26 @@ def channels_to_send(self, digest=False):
         return channels
 
     def send_if_required(self):
-        from app.modules.emails.utils import EmailUtils
+        from app.extensions.email import Email
 
         channels = self.channels_to_send(False)
 
+        self._channels_sent = {}  # store what was sent out, if anything
         if channels[NotificationChannel.email]:
             config = NOTIFICATION_CONFIG[self.message_type]
             email_message_values = {
                 'context_name': 'context not set',
+                'sender_name': self.recipient.full_name,
```

Comment:
hang on, that's the recipient full name, not tthe sender...

Reply:
oopsy!  fixed.